docs/dialer.rst

```diff
@@ -205,10 +205,6 @@ If you want to write your own Pymess dialer backend you must create class that i
 
     This method should send dialer message (obtained from the input argument) and update its state. This method must be overridden in the custom backend.
 
-  .. method:: publish_messages(messages)
```

Comment:
This is the main issue I encountered with dynamic backends. A lot of methods should be actually called outside of certain backend and inside the function the backend would be chosen from backend type and recipient. You will see what I mean.
Also this would change lib API.
Methods `create_message`, `publish_messages` and `bulk_send` needed to be refactored (they are called from our core project)
Once we find the proper solution I will change the docs accordingly.